bin/sozo/src/ops/migration/mod.rs

```diff
@@ -486,7 +486,9 @@ where
 
     TransactionWaiter::new(transaction_hash, migrator.provider()).await?;
 
-    ui.print_sub(format!("Registered at: {transaction_hash:#x}"));
+    ui.print_header(format!("======================================================"));
+
+    ui.print_sub(format!("All models are registered at: {transaction_hash:#x}"));
```

Comment:
Could we change that for a simple print? To. It have it indented as the model declaration?

Reply:
do you mean line 491? or 489?